django_project/trail/models/grade.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+"""Model definitions for a trail app"""
+
+import os
+import uuid as uuid_lib
+
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.utils.text import slugify
+from django.conf.global_settings import MEDIA_ROOT
+
+from unidecode import unidecode
+
+from core.settings.contrib import STOP_WORDS
+
+
+__author__ = 'Alison Mukoma <alison@kartoza.com>'
+__date__ = '01/10/2018'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+
+def increment_slug(_name):
```

Comment:
This looks almost identical to the one above. Please normalize the function, move this to an utilities lib and import it from there.